resources/content/schema/base/container.json

```diff
@@ -187,5 +187,11 @@
             "type":"array[string]",
             "nullable": true
         }
+    },
```

Comment:
Thought it had to be in both places sorry. Hadn't caught the dups as when i did some of this i was still learning your frame work. You have multiple places to put things to make your magic happen.